### ISSUE TITLE
Hugging Face Error

### ISSUE THREAD

**Issue by ranjancse26** [limcheekin/open-text-embeddings#10]
Looks like the hugging face space no longer exist. https://[limcheekin-bge-small-en-v1-5.hf.space/v1](https://limcheekin-bge-small-en-v1-5.hf.space/v1)

Receiving the following error

Your space is on error, check its status on hf.co

**Comment by limcheekin** [comment]
Thanks for report the issue.

I fixed the `HF_HOME` environment variable error. Next, it return `500 Internal Server Error` with the following error trace:
```
    | Traceback (most recent call last):
    |   File "/usr/local/lib/python3.11/site-packages/uvicorn/protocols/http/h11_impl.py", line 408, in run_asgi
    |     result = await app(  # type: ignore[func-returns-value]
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/local/lib/python3.11/site-packages/uvicorn/middleware/proxy_headers.py", line 84, in __call__
    |     return await self.app(scope, receive, send)
    |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   File "/usr/local/lib/python3.11/site-packages/fastapi/applications.py", line 1054, in __call__
    |     await super().__call__(scope, receive, send)
    |   File "/usr/local/lib/python3.11/site-packages/starlette/applications.py", line 116, in __call__
    |     await self.middleware_stack(scope, receive, send)
    |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 186, in __call__
    |     raise exc
    |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/errors.py", line 164, in __call__
    |     await self.app(scope, receive, _send)
    |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/gzip.py", line 24, in __call__
    |     await responder(scope, receive, send)
    |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/gzip.py", line 44, in __call__
    |     await self.app(scope, receive, self.send_with_gzip)
    |   File "/usr/local/lib/python3.11/site-packages/starlette/middleware/base.py", line 189, in __call__
    |     with collapse_excgroups():
    |   File "/usr/local/lib/python3.11/contextlib.py", line 158, in __exit__
    |     self.gen.throw(typ, value, traceback)
```

I think the error is HF space specific as it is not happened in AWS Lambda function. 

As I am busy on other stuff now, please help to look into it if you are urgent. Just duplicate the space and comment out the code at https://github.com/limcheekin/open-text-embeddings/blob/main/open/text/embeddings/server/app.py#L43 to see whether it fix the error (You need to modify the Dockfile to use the code instead of the `open-text-embeddings` package).

**Comment by limcheekin** [comment]
Fixed in release 1.0.4
